.github/workflows/pythonbuild.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           if [[ ${{ github.event_name }} == "schedule" ]]; then
             echo "python_versions=[\"3.8\",\"3.9\",\"3.10\",\"3.11\",\"3.12\"]" >> $GITHUB_ENV
           else
-            echo "python_versions=[\"3.12\"]" >> $GITHUB_ENV
+            echo "python_versions=[\"3.8\", \"3.12\"]" >> $GITHUB_ENV
```

Comment:
python 3.8 is hitting EOL in November according to https://devguide.python.org/versions/#supported-versions. What do you think of using 3.9 instead (in preparation for the removal of python 3.8)?

Reply:
I'll update it later

Reply:
updated